fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -345,20 +338,18 @@ private Map<PartitionCacheKey, HivePartition> loadPartitions(Iterable<? extends
     }
 
     // Get File Status by using FileSystem API.
-    private FileCacheValue getFileCache(String location, String inputFormat,
-            JobConf jobConf,
-            List<String> partitionValues,
-            String bindBrokerName,
-            DirectoryLister directoryLister,
-            TableIf table) throws UserException {
+    private FileCacheValue getFileCache(LocationPath path, String inputFormat,
+                                        List<String> partitionValues,
+                                        DirectoryLister directoryLister,
+                                        TableIf table) throws UserException {
         FileCacheValue result = new FileCacheValue();
-        Map<String, String> properties = catalog.getCatalogProperty().getProperties();
-        RemoteFileSystem fs = Env.getCurrentEnv().getExtMetaCacheMgr().getFsCache().getRemoteFileSystem(
-                new FileSystemCache.FileSystemCacheKey(LocationPath.getFSIdentity(
-                        location, properties, bindBrokerName),
-                        properties,
-                        bindBrokerName, jobConf));
-        result.setSplittable(HiveUtil.isSplittable(fs, inputFormat, location));
+
+        FileSystemCache.FileSystemCacheKey fileSystemCacheKey = new FileSystemCache.FileSystemCacheKey(
+                path.getFsIdentifier(), path.getStorageProperties()
```

Comment:
I think this is not a good idea to get storage properties in `LocationPath`,
And you also save the storage properties in `FileSystemCacheKey`, which may led to some problem